src/dc_receive_imf.rs

```diff
@@ -1085,17 +1065,11 @@ INSERT INTO msgs
     ?, ?, ?, ?,
     ?, ?, ?, ?,
     ?, ?, ?, ?,
-    ?, ?
+    ?
   );
 "#,
         )?;
 
-        if is_mdn
-            || (location_kml_is && icnt == 1 && (part.msg == "-location-" || part.msg.is_empty()))
-        {
-            is_hidden = true;
-        }
```

Comment:
not sure if that check can really go away - it is used to not show the location-only messages (the ones that are send every some minutes) - or are they sorted out before somehow?

the `-location-` body is indeed removed, so, if they do no result in an empty part, things might be okay.

Reply:
`do_add_single_file_part` in `mimeparser.rs` already skips these parts, saving them only in `MimeMessage.location_kml`.

Reply:
k, thanks for double checking!